### DISCUSSION TITLE
Default value on CRUD collection field

### DISCUSSION THREAD

**Discussion by calairoh** [mia-platform/community#147] [Q&A] [answered]
Hi all!
Is it possible to set a default value for a property in a CRUD collection?

**Comment by fredmaggiowski** [comment, marked as answer]
Hi no, right now it isn't; it may be an interesting feature.  @nicocampagna what do you think?

**Reply by TheRusCompass** [reply]
Thank you for the suggestion @calairoh!! That would be an interesting feature, we will consider to add it to the roadmap

**Reply by calairoh** [reply]
I would add that it might be interesting if you could set up a function to calculate the default value for that specific property, as well as a constant value.
